### PR TITLE
update backward disks position, and reduce disk inner radii.

### PR DESCRIPTION
**Pull request by ShujieL**
### Briefly, what does this PR introduce?

Update backward silicon disk position according to Ernst's re-optimization at https://indico.bnl.gov/event/17418/contributions/69075/attachments/43717/73653/20221010%20-%20EIC%20silicon%20consortium%20geometry.pdf

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [x] New feature (issue #_177_)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
This change will create temporarily inconsistence between the tracking geometry and material map, until the new material map is produced

### CODE REVIEW

File: compact/tracker.xml [hunk 1]
```diff
@@ -71,13 +71,13 @@
...
     <constant name="TrackerEndcapNDisk1_rmin"        value="TrackerEndcapPDisk1_rmin" />
     <constant name="TrackerEndcapNDisk1_rmax"        value="TrackerEndcapPDisk1_rmax" />
-    <constant name="TrackerEndcapNDisk2_zmin"        value="TrackerEndcapPDisk2_zmin" />
+    <constant name="TrackerEndcapNDisk2_zmin"        value="65*cm" />
     <constant name="TrackerEndcapNDisk2_rmin"        value="TrackerEndcapPDisk2_rmin" />
     <constant name="TrackerEndcapNDisk2_rmax"        value="TrackerEndcapPDisk2_rmax" />
-    <constant name="TrackerEndcapNDisk3_zmin"        value="TrackerEndcapPDisk3_zmin" />
+    <constant name="TrackerEndcapNDisk3_zmin"        value="90*cm" />
     <constant name="TrackerEndcapNDisk3_rmin"        value="TrackerEndcapPDisk3_rmin" />
     <constant name="TrackerEndcapNDisk3_rmax"        value="TrackerEndcapPDisk3_rmax" />
```
Comment (wdconinc) [inline compact/tracker.xml:79]: Will the rmin (and rmax) stay the same even if you change the zmin?

Comment (ShujieL) [reply compact/tracker.xml:79]: For now I don't have beampipe radii at those new location so I kept the previous number. So those three disks at electron direction may have a few milimeter larger than necessary opening around the beampipe but should not break anything.

Comment (ShujieL) [reply compact/tracker.xml]: I updated rmin to reflect the new z positions.

**Review by osbornjd — COMMENTED** [review]
Looks good to me from an integration standpoint. We could consider parameterizing the radius based on the z position so that it is automatically updated, but that is further development that can happen after the campaign.

File: compact/tracker.xml [hunk 2]
```diff
@@ -71,17 +74,17 @@
...
+    <constant name="TrackerEndcapNDisk3_rmin"        value="34.244*mm + Beampipe_bakeout_buffer + 0.817*mm" />
     <constant name="TrackerEndcapNDisk3_rmax"        value="TrackerEndcapPDisk3_rmax" />
-    <constant name="TrackerEndcapNDisk4_zmin"        value="TrackerEndcapPDisk4_zmin" />
-    <constant name="TrackerEndcapNDisk4_rmin"        value="TrackerEndcapPDisk4_rmin" />
+    <constant name="TrackerEndcapNDisk4_zmin"        value="115*cm" />
+    <constant name="TrackerEndcapNDisk4_rmin"        value="38.043*mm + Beampipe_bakeout_buffer + 3.310*mm" />
     <constant name="TrackerEndcapNDisk4_rmax"        value="TrackerEndcapPDisk4_rmax" />
   </define>
-
+
```
Comment (wdconinc) [inline compact/tracker.xml:87]:
```suggestion
```

**Review by wdconinc — APPROVED** [review]

**Comment by ShujieL** [comment]
This PR fails some reconstruction benchmarks but those should be irrelevant?

**Pull request merged**